libs/markets/src/lib/components/markets-container/use-column-defs.tsx

```diff
@@ -181,6 +164,79 @@ export const useColumnDefs = ({ onMarketClick }: Props) => {
             );
           },
         },
+        {
+          headerName: t('Spread'),
+          field: 'data.bestBidPrice',
+          type: 'rightAligned',
+          filter: 'agNumberColumnFilter',
+          valueGetter: ({
+            data,
+          }: VegaValueGetterParams<MarketMaybeWithData>) => {
+            const offer =
+              data?.data?.bestOfferPrice === undefined
+                ? undefined
+                : toBigNum(
+                    data?.data?.bestOfferPrice,
+                    data.decimalPlaces
+                  ).toNumber();
+            const bid =
+              data?.data?.bestBidPrice === undefined
+                ? undefined
+                : toBigNum(
+                    data?.data?.bestBidPrice,
+                    data.decimalPlaces
+                  ).toNumber();
+            return Number(bid) - Number(offer);
+          },
+          cellRenderer: ({
```

Comment:
The spread column should render the actual calculated spread. Not the best offer/ask next to each other